### DISCUSSION TITLE
Tailwindcss colors randomly not working anymore???

### DISCUSSION THREAD

**Discussion by ma-a-sc** [tailwindlabs/tailwindcss#9162] [Help]
Hello,

I started experimenting with TailwindCss yesterday using FastApi as the backend.
It worked fine I was following some tutorials on how to get started closed my Laptop for the night and when I wanted to start today adding color didn't work. The container class or mx-auto worked fine or other things like fonts. So its not that my html files are not picked up by the content in the config file. 
Color would not apply and even more interesting the color wich worked fine yesterday evening is "removed" (does not apply anymore) if I only e.g. use bg-emerald-200 instead of bg-emeral-400.
I cant figure out what has happened and I can't seem to find anything online.

Does somebody have an idea what could be going on?
Tailwind builds fine and uvicorn also does not report any errors.
I did not change the code base in any meaningful way (at least to my knowledge). The config file is the same as it is yesterday. I did not accidentally override the colors or something similar. I am completely lost.

Thanks in advance and kind regards from Germany.

**Comment by wongjn** [comment]
Could you supply a repository that reproduces the problem?
